packages/flame/lib/src/extensions/vector2.dart

```diff
@@ -93,6 +95,9 @@ extension Vector2Extension on Vector2 {
     }
   }
 
+  /// Signed angle in a coordinate system where the Y-axis is flipped.
```

Comment:
can we give more details of precisely how this system is defined? and maybe compare it with the normal angle() function explaining the difference?

Reply:
Done! And added tests for it.